RecursiveExtractor/ArFile.cs

```diff
@@ -89,7 +89,7 @@ public static IEnumerable<FileEntry> GetFileEntries(FileEntry fileEntry, Extract
                             // The name length is included in the total size reported in the header
                             CopyStreamBytes(fileEntry.Content, entryStream, size - nameLength);
 
-                            yield return new FileEntry(Encoding.ASCII.GetString(nameSpan).TrimEnd('/'), entryStream, fileEntry, true);
+                            yield return new FileEntry(Encoding.ASCII.GetString(nameSpan).TrimEnd('/'), entryStream, fileEntry, true, memoryStreamCutoff: options.MemoryStreamCutoff);
```

Comment:
We should probably go back and make sure we're not being overly 'ASCII', since most filesystems AFAICT support UTF-8 / Unicode filenames.

Reply:
In the AR Format it must be ascii